Sources/SafeDIMacros/Macros/InstantiableMacro.swift

```diff
@@ -106,20 +106,29 @@ public struct InstantiableMacro: MemberMacro {
                 .contains(where: { $0.isValid(forFulfilling: visitor.dependencies) })
             guard hasMemberwiseInitializerForInjectableProperties else {
                 if visitor.uninitializedNonOptionalPropertyNames.isEmpty {
-                    var initializer = Initializer.generateRequiredInitializer(
```

Comment:
I recommend viewing this file in side-by-side mode – this code mostly moved around rather than changed